groovy/postInit/chemistry/elements/AluminiumChain.groovy

```diff
@@ -14,6 +14,14 @@ CRYSTALLIZER = recipemap('crystallizer')
 
 // Aluminium Nugget * 3
 mods.gregtech.electric_blast_furnace.removeByInput(100, [metaitem('dustRuby')], null)
+//Aluminium Dust -> Aluminium Ingot
```

Comment:
remove this section

Reply:
why?
considering the only way to get aluminium dust
is smelting aluminium-containing machine